timetracker/.idea/.name

```diff
@@ -0,0 +1 @@
+timetracker
```

Comment:
Looks like you've possibly done a "git add -a" or similar, and added files that really don't belong in your repo.

Remember: explicit is better than implicit.